COMPATIBILITY.md

```diff
@@ -13,6 +13,7 @@ See [this guide](scm-step/README.md#supporting-workflow-from-an-scm-plugin) for
 - [X] `MercurialSCM` (`mercurial`): supported as of 1.51
 - [ ] `PerforceScm` (`p4`, not the older `perforce`): [JENKINS-24206](https://issues.jenkins-ci.org/browse/JENKINS-24206)
 - [ ] `DimensionsSCM` (`dimensionsscm`): [JENKINS-26165](https://issues.jenkins-ci.org/browse/JENKINS-26165)
+- [ ] `SubversionTagging` (`svn-tag`): [JENKINS-26949](https://issues.jenkins-ci.org/browse/JENKINS-26949)
```

Comment:
Ok I've moved it, and based on the understanding this is now the class name not the plugin name, I've changed it to SvnTagPublisher